lib/fhir_models.rb

```diff
@@ -1,44 +1,36 @@
-# Top level include file that brings in all the necessary code
-require 'bundler/setup'
+require 'fhir_models/version'
+
+# TODO what are these for?  Nothing seems to use them.
 require 'yaml'
-require 'nokogiri'
 require 'date_time_precision'
 require 'date_time_precision/format/iso8601'
-require 'mime/types'
-require 'bcp47'
-require 'bigdecimal'
-require 'logger'
-require 'tempfile'
-
-root = File.expand_path '..', File.dirname(File.absolute_path(__FILE__))
 
-# Need to require Hashable first
-require File.join(root,'lib','bootstrap','hashable.rb')
-require File.join(root,'lib','bootstrap','json.rb')
+Dir.chdir('lib') do
+  # Need to require Hashable first
+  require_relative 'fhir_models/bootstrap/hashable'
+  require_relative 'fhir_models/bootstrap/json'
+  Dir['fhir_models/bootstrap/*.rb'].each do |file|
+    require_relative file
+  end
 
-Dir.glob(File.join(root, 'lib','bootstrap','*.rb')).each do |file|
-  require file
-end
-Dir.glob(File.join(root, 'lib','bootstrap','**','*.rb')).each do |file|
-  require file
-end
+  require_relative 'fhir_models/fhir'
 
-require File.join(root, 'lib','fhir.rb')
+  # require generated models
+  Dir['fhir_models/fhir/**/*.rb'].each do |file|
+    require_relative file
+  end
 
-# Require the generated code
-Dir.glob(File.join(root, 'lib','fhir','*.rb')).each do |file|
-  require file
-end
-Dir.glob(File.join(root, 'lib','fhir','**','*.rb')).each do |file|
-  require file
-end
+  # require fluentpath code
+  Dir['fhir_models/fluentpath/*.rb'].each do |file|
+    require_relative file
+  end
 
-# Require the fluentpath code
-Dir.glob(File.join(root, 'lib','fluentpath','*.rb')).each do |file|
-  require file
+  # require fhir_ext code
+  Dir['fhir_models/fhir_ext/*.rb'].each do |file|
+    require_relative file
+  end
 end
 
-# Require the fhir_ext code
-Dir.glob(File.join(root, 'lib','fhir_ext','*.rb')).each do |file|
-  require file
+module FhirModels
```

Comment:
oops, I guess we don't really need this since we aren't using the `FhirModels` namespace.